src/App.vue

```diff
@@ -227,6 +227,22 @@ export default {
 		},
 	},
 
+	beforeCreate() {
+		const authorizedUser = getCurrentUser()?.uid || null
+		const lastLoggedInUser = BrowserStorage.getItem('last_logged_in_user')
+
+		if (authorizedUser !== lastLoggedInUser) {
```

Comment:
Isn't it still comparing `null !== 'null'`?

Reply:
We don't save uid for guests, so as long as there is nothing saved, `BrowserStorage.getItem()` returns normal `null`